recipe/meta.yaml

```diff
@@ -1,12 +1,17 @@
-{% set version = "1.6.3" %}
+{% set version = "1.7.0rc1" %}
 
 package:
   name: scipy
   version: {{ version }}
 
 source:
-  url: https://github.com/scipy/scipy/archive/v{{ version }}.tar.gz
-  sha256: 6cdb9d3aabe46c1602d9fd2f34812700068eddfe3ffd1dafcbd7e7960a874e1c
+  - url: https://github.com/scipy/scipy/archive/v{{ version }}.tar.gz
```

Comment:
Picking the GitHub-generated tarball seems unusual to me. The actual sdist is also there: https://github.com/scipy/scipy/releases/download/v1.7.0rc1/scipy-1.7.0rc1.tar.gz

Reply:
Well, that's just scipy being a exemplary FOSS citizen 😅 - most libraries don't have more than the auto-generated GH tarballs (which are still considered an authoritative source). In any case, assuming that sdist is the same as what was uploaded to PyPI, that would fail with the same problems as above. But I don't mind switching to this link for rc2.